data/language/english_uk.txt

```diff
@@ -4039,8 +4038,20 @@ STR_5732    :Anisotropic
 STR_5733    :Use NN scaling at integer scales
 # tooltip for tab in options window
 STR_5734    :{SMALLFONT}{BLACK}Rendering
-STR_5735    :Network Status
-STR_5736    :Player
```

Comment:
@thyeggman yeah we know it is annoying but it doesn't happen that often and it doesn't exactly take too long to change. There was talk a little while ago about reserving numbers which you can do by just doing a quick PR with which ones you want reserved but I haven't seen anyone do that for quite a while.